fabric/src/main/java/gg/moonflower/pollen/core/mixin/fabric/ExplosionMixin.java

```diff
@@ -0,0 +1,27 @@
+package gg.moonflower.pollen.core.mixin.fabric;
+
+import gg.moonflower.pollen.api.event.events.world.ExplosionEvents;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.level.Explosion;
+import net.minecraft.world.level.Level;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+
+import java.util.List;
+import java.util.Set;
+
+@Mixin(Explosion.class)
+public abstract class ExplosionMixin {
+    @Accessor("level")
+    abstract Level getLevel();
```

Comment:
This doesn't need to be an accessor, you should just access the field using ``@Shadow``

Reply:
Changed in latest commit.